### PR TITLE
Refactoring code in plugin-tester-java(scala)

### PR DESCRIPTION
**Pull request by laglangyue**
## motivation
There are some unnecessary variable and style in `plugin-tester-Java` that does not use variables. I have readed the code for these two modules. clean up the code
plugin-tester-scala.
BTW, PowerGreeterServer and PowerGreeterServer lost the client, it will be an another PR later.

## result
clean up

### CODE REVIEW

File: plugin-tester-scala/src/test/scala/example/myapp/helloworld/GreeterServiceSpec.scala [hunk 1]
```diff
@@ -31,7 +31,7 @@ import scala.concurrent.{ Await, ExecutionContext }
 import scala.concurrent.duration._
 
 @RunWith(classOf[JUnitRunner])
-class GreeterSpec extends Matchers with AnyWordSpecLike with BeforeAndAfterAll with ScalaFutures {
```
Comment (laglangyue) [inline plugin-tester-scala/src/test/scala/example/myapp/helloworld/GreeterServiceSpec.scala:34]: the fileName is GreeterServiceSpec in scala module. And in java moudle, it's JGreeterServiceSpec. So use the name

**Review by pjfanning — CHANGES_REQUESTED** [review]
Breaks the build - can this be fixed?

```
[error] :  @@snip [GreeterServicePowerApiImpl.java](/plugin-tester-java/src/main/java/example/myapp/helloworld/GreeterServicePowerApiImpl.java) { #full-service-impl }
[error]    ^
```

**Comment by laglangyue** [comment]
@pjfanning @He-Pin @raboof PTAL

**Review by He-Pin — APPROVED** [review]
Lgtm, thanks

**Review by He-Pin — APPROVED** [review]

File: plugin-tester-java/src/main/java/example/myapp/helloworld/LiftedGreeterClient.java [hunk 2]
```diff
@@ -68,7 +65,7 @@ private static void singleRequestReply(GreeterServiceClient client) throws Excep
 
   private static void streamingRequest(GreeterServiceClient client) throws Exception {
     List<HelloRequest> requests =
-        Arrays.asList("Alice", "Bob", "Peter").stream()
+        Stream.of("Alice", "Bob", "Peter")
```
Comment (He-Pin) [inline plugin-tester-java/src/main/java/example/myapp/helloworld/LiftedGreeterClient.java:68]: Elegant

**Review by He-Pin — APPROVED** [review]

**Comment by laglangyue** [comment]
@pjfanning PTAL again. Thanks

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**